runtime-light/stdlib/fork/fork-context.h

```diff
@@ -17,19 +17,29 @@ class ForkComponentContext {
   template<hashable Key, typename Value>
   using unordered_map = memory_resource::stl::unordered_map<Key, Value, memory_resource::unsynchronized_pool_resource>;
 
-  static constexpr auto FORK_ID_INIT = 1;
+  static constexpr auto FORK_ID_INIT = 0;
 
   unordered_map<int64_t, task_t<fork_result>> forks_;
   int64_t next_fork_id_{FORK_ID_INIT};
+  ino64_t running_fork_id_{FORK_ID_INIT};
 
 public:
   explicit ForkComponentContext(memory_resource::unsynchronized_pool_resource &memory_resource) noexcept
     : forks_(unordered_map<int64_t, task_t<fork_result>>::allocator_type{memory_resource}) {}
 
   static ForkComponentContext &get() noexcept;
 
+  int64_t get_running_fork_id() const noexcept {
```

Comment:
Actually, I don't see any benefits of having such methods. We don't have any logic that should wrap access/modify of current fork id. So let's just have a public member `current_fork_id`

Reply:
Can't agree. I don't want to add `php_debug(...)` in all places where code modify current_fork_id.

Reply:
Your approach has several drawbacks:

1. it's a responsibility of `ForkComponentContext` to determine what fork is currently running. `ForkComponentContext` is just a storage to save current fork id and all forks that were started and not yet awaited by some other fork;
2. putting logs into `ForkComponentContext` you actually loose log accuracy. For example, you'll get unnecessary logs in case of awaitable that is `await_ready`.

Reply:
ok